src/System.Linq.Expressions/tests/Unary/UnaryQuoteTests.cs

```diff
@@ -3,16 +3,17 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Reflection;
+using System.Runtime.CompilerServices;
 using Xunit;
 
 using static System.Linq.Expressions.Expression;
 
 namespace System.Linq.Expressions.Tests
 {
-    public class UnaryQuoteTests
+    public static class UnaryQuoteTests
```

Comment:
I'll revert that change and we can look into making the tests follow the recommendation consistently in a separate PR.